prepare_podman_compose.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# make sure selinux is off
+sestatus | grep '^Current mode:.*enforcing$'
+if [ $? -eq 0 ]; then
+  echo "Selinux is enforcing.Set it to permissive mode before using podman-compose"
+  exit 1
+fi
+
+mkdir -p ./tmp/cachito-archives
```

Comment:
Apparently this is an explicit backwards compatibility breakage, see https://github.com/containers/podman-compose/issues/185

It looks like a compatibility flag could be introduced to handle this.

Reply:
Will this podman-compose patch also eliminate the need to change the nexus-volume permissions below?

Reply:
Hmm unclear, but I was able to share named volumes between two containers in iib.